pkg/services/authn/clients/ldap.go

```diff
@@ -131,4 +131,12 @@ func (c *LDAP) identityFromLDAPInfo(orgID int64, info *login.ExternalUserInfo) *
 			},
 		},
 	}
+
+	// The ldap service is not aware of the internal state of the user. Fetching the user
+	// from the store to know if that user is disabled or not, is almost as costly as
+	// running an update systematically. We are setting IsDisabled to true so that the
+	// EnableDisabledUserHook force-enable that user.
+	id.IsDisabled = true
```

Comment:
Only edge case I can think of is if, in the future, we introduce a hook before the `EnableDisabledUserHook` (priority `20`) and that hook relies on the `IsDisabled` value as well.

Reply:
But that would be strange to place a hook using `IsDisabled` before the `EnableDisabledUserHook`